src/Menu/page.tsx

```diff
@@ -12,6 +15,43 @@ export default function MenuPage() {
         heading
       </Heading>
       <Blockquote weight="bold">quote quote</Blockquote>
+
+      <UserListContainer>
+        <Flex direction="column" gap="2">
```

Comment:
* Flex 정렬을 radix 컴포넌트 활용해서 구현하려면 이렇게 스타일 오버라이드로 사용해도 좋을것 같아요! 
  * 현재 구조면 정렬을 위해 한 번 더 depth가 들어가게 되는 구조인것 같은데, 개인적으로 생각했을 때 최대한 태그 depth를 줄이면 좋을것 같다는 의견이 있습니둥

* 다만 이렇게 사용하면 display 로직이 직관적으로 들어나지 않아 한 눈에 코드가 잘 읽히지 않는것 같아요 (flex라는 로직은 하위 스타일드 컴포넌트 정의에, direction이나 gap 관련 로직은 jsx 태그에 표현되어 정렬 관련 로직이 흩어져 있어서)
-> 이런 측면에서 생각해봤을 때 flex와 같은 정렬 로직은 radix 컴포넌트 사용하지 않고, 개별 스타일로 적용해줘도 좋을것 같은데 요거 어떻게 생각하는지 서현이 정우 의견 궁금합니듀 ~


```suggestion
      <UserListContainer direction="column" gap="10px">
      
      ...
      
      const UserListContainer = styled(Flex)`
  width: "100vw";
  padding: "10px";
`;

```

Reply:
@Yeonseo-Jo
Flex 컴포넌트 스타일 오버라이드하는 것 관련해서 저는 Flex 컴포넌트의 목적상 flex에 관한 책임만 가져야 한다고 생각했어요!! 그런데 연서 말대로 depth가 깊어지기 때문에 스타일 오버라이드도 괜찮다고 생각하는데, 그렇다면 Radix의 Flex 컴포넌트를 오버라이드했다는 것을 명시하기 위해 스타일 컴포넌트 네이밍에 `Flex`가 포함되는 게 좋다고 생각해요!!

사실 저도 스타일을 담당하는 부분이 흩어져 있다는 생각이 들어서, 이런 측면에서는 Flex를 오버라이딩하기보다는 시맨틱 태그로 스타일 컴포넌트를 만드는게 더 좋아보이기도 하네요!! 물론 Radix를 최대한 활용해보자는 의의와는 좀 멀어지긴 하겠지만요...!

Reply:
이 부분 마침 회의에서 얘기 나눠보고 싶은 부분이었는데요!
전반적으로 저희가 스타일링 관련해서 어떻게 코드 스타일을 가져갈지 논의해보고 정하면 좋을 것 같아요.

1. Flex, Grid, Heading, Text 등 Radix 컴포넌트를 최대한 사용해 스타일도 선언적으로 사용하기
- 이때 스타일 오버라이드는 어떤 방식으로 할 것인가
    - styled(Flex) -> `~~Flex` 라는 이름으로 통일하기와 같은 컨벤션
    - css props 이용하기
- direction, gap 과 같은 props를 적극 사용 or not

2. Radix 컴포넌트 보다는 styled로 개별 스타일 적용해서 사용하기

3. 저희가 전부 직접 스타일링해서 컴포넌트를 만드는게 아니라 Radix를 많이 이용할 예정이다 보니, 과연 styled만 사용하는게 DX 측면에서 옳은 선택일지도 논의해보고 싶어요! css prop으로 스타일을 주는 것이 꽤 유용한 상황이라고 생각되었거든요,,

이 주제 다음 회의에서 얘기 나눠봅시당